tests/NLog.UnitTests/Targets/FileTargetTests.cs

```diff
@@ -2280,14 +2282,21 @@ public void Dont_throw_Exception_when_archiving_is_enabled()
 </nlog>
 ");
 
-            NLog.LogManager.GetLogger("Test").Info("very important message");
+                NLog.LogManager.GetLogger("Test").Info("very important message");
+            }
+            finally
+            {
+                LogManager.Configuration = null;
```

Comment:
I tried something slimiar, but  in the ctor of NLogTestBase (as xUnit create a new instance of the test class every test)

https://github.com/NLog/NLog/blob/master/tests/NLog.UnitTests/NLogTestBase.cs#L64

So any idea why that won't work?